src/utilities/speedStatMapper.ts

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable @typescript-eslint/no-magic-numbers */
+const MINIMUM_SPEED_STAT = 380
+const BASE_GCD = 2500
+
+const STAT_DIVISOR = 3300
+
+export function getSpeedStat(estimatedGcd: number): number {
```

Comment:
i _wanna_ say that this should be an impl detail of the adapter - by estimating the attr value in the adapter phase, we're stating that the concept of an actor's attrs is a log-concern - ergo, there should be no need to reverse a gcd to a spks value elsewhere, only spks -> recast

---

should probably extend speed -> recast to support the other variables in the equation (that we're currently simplifying out) so that we're able to support speed modifiers at parse time.